bot/resources/tags/json.md

```diff
@@ -0,0 +1,27 @@
+When creating a new JSON file you might run into the following error.
+
+`JSONDecodeError: Expecting value: line 1 column 1 (char 0)`
+
+In short this error means your JSON is invalid in it's current state.
+A JSON may never be completely empty and must always at least have one of the following items.
```

Comment:
I'd lead into this more following the previous statement e.g. "This is probably because your new file is empty."